probedata.go

```diff
@@ -102,6 +103,35 @@ type StreamDisposition struct {
 	AttachedPic     int `json:"attached_pic"`
 }
 
+// Chapters is a json data structure to represent chapters.
+type Chapter struct {
+	ID               int         `json:"id"`
+	TimeBase         string      `json:"time_base"`
+	StartTimeSeconds float64     `json:"start_time,string"`
+	EndTimeSeconds   float64     `json:"end_time,string"`
+	Tags             *ChapterTags `json:"tags"`
```

Comment:
If this is just a mapping, cant we re-use the `Tags` type, saving us from having to add all props separately?